src/utils/api.js

```diff
@@ -4,8 +4,16 @@ const newsApi = axios.create({
   baseURL: "https://nc-news-2yza.onrender.com/api",
 });
 
-export const getAllArticles = () => {
-  return newsApi.get("/articles").then(({ data }) => {
-    return data.articles;
+export const getArticles = (article_id) => {
```

Comment:
I would have extracted these out into two separate functions (getArticles, getArticleById), only in case the separate endpoints accept different queries later on.